Update AgentRunnerThread.h

Cr4shOv3rrid3 opened the pull request:
fixed typo line 96

Cr4shOv3rrid3 committed:
Update AgentRunnerThread.h

fixed typo

linas commented:
You are fixing typos in code that is 9 years out of date, and nine thousand commits behind master. If you are serious, please review the repos at https://github.com/opencog/

Cr4shOv3rrid3 committed:
Update NetworkServer.h

typo + unnecessary empty line

Cr4shOv3rrid3 committed:
Update NetworkServer.cc

line 104 i don't understand how an avoiding should make sense here when it adds a wait anyways exactly by implementing usleep() function. so it should be declared add and not avoid

linas commented:
Again: you are making changes to code that has long ago been abandoned. The present-day version of the cogserver, which has the `NetworkServer.cc|h` files that you are making changes to, can be found at https://github.com/opencog/cogserver

linas commented:
@Cr4shOv3rrid3 if you wish to have a general conversation about the status of things, the projects, the future directions, write to me.

Cr4shOv3rrid3 committed:
Update ConsoleSocket.cc

0xff should be only for IAC not for CHARSET so this needs a fix into 0x2a (according to variable definition for CHARSET before in code)

Cr4shOv3rrid3 commented:
> You are fixing typos in code that is 9 years out of date, and nine thousand commits behind master. If you are serious, please review the repos at https://github.com/opencog/

i wasnt aware of the fact that this own repo also exists. i developed from bitcoin direction and this also was first coded in 2008. so i came into hansonrobotics repo first, gonna give the continued opencog isolated repo a chance then. will let this fork keep intact however cause i am not 100% sure at the moment what is best decision. still have to work into this a lot first. so its reading time and should i found new typos or so i am happy to fix them for you guys too! gives a bit to do besides reading anyways. so i wont get bored ;)

linas commented:
I worked at Hanson Robotics, I put a lot of this code into place, I split it up into pieces to make it easier to manage and to work with. Many people dozens, and then some more, contributed to the code base to clean it up and simplify it and make it suitable for general use.

If you have trouble building all the pieces, you might want to explore https://github.com/opencog/docker which, among other things, includes the (old) containers used by actual live running hanson robotics systems.
